anomalib/data/base/datamodule.py

```diff
@@ -101,8 +115,20 @@ def train_dataloader(self) -> TRAIN_DATALOADERS:
 
     def val_dataloader(self) -> EVAL_DATALOADERS:
         """Get validation dataloader."""
-        return DataLoader(self.val_data, shuffle=False, batch_size=self.eval_batch_size, num_workers=self.num_workers)
+        return DataLoader(
+            self.val_data,
+            shuffle=False,
+            batch_size=self.eval_batch_size,
+            num_workers=self.num_workers,
+            collate_fn=collate_fn,
+        )
```

Comment:
Sorry, it's a bit picky comment, but would be be easier to read if all the arguments are specified.
```suggestion
        return DataLoader(
            dataset=self.val_data,
            shuffle=False,
            batch_size=self.eval_batch_size,
            num_workers=self.num_workers,
            collate_fn=collate_fn,
        )
```